exec/exec_test.go

```diff
@@ -86,3 +87,12 @@ func TestBadReturnCode(t *testing.T) {
 		t.Fatalf("Expected `command_no_found` in error output.Got: %s", err)
 	}
 }
+
+func TestOutput(t *testing.T) {
+
+	out, stderr, err := Output(nil, 0, "echo", "hello")
+	if err != nil {
+		t.Fatal(err)
+	}
+	fmt.Printf("output %s %s \n", out, stderr)
+}
```

Comment:
could we run a more command that write to both stdout and stderr and then validates the result instead of just checking for error?